docs/modeling/index.rst

```diff
@@ -322,6 +322,30 @@ are some complexities involved in correctly matching up the inputs and outputs
 of all models used to build a compound model.  You can learn more details in
 the :doc:`compound-models` documentation.
 
+Astropy models also support convolution through the function
+`~astropy.convolution.convolve_models`, which returns a compound model.
+
+For instance, the convolution of two Gaussian functions is also a Gaussian
+function in which the resulting mean (variance) is the sum of the means
+(variances) of each Gaussian.
+
+.. plot::
+    :include-source:
+
+    import numpy as np
+    import matplotlib.pyplot as plt
+    from astropy.modeling import models
+    from astropy.convolution import convolve_models
+
+    g1 = models.Gaussian1D(1, -1, 1)
+    g2 = models.Gaussian1D(1, 1, 1)
+    g3 = convolve_models(g1, g2)
+
+    x = np.linspace(-3, -3, 50)
+    plt.plot(x, g1(x), 'k-')
+    plt.plot(x, g2(x), 'k-')
+    plt.plot(x, g3(x), 'k-')
+
```

Comment:
Currently the user has to take care of normalization, centering, fixing of parameters him- or herself, which is totally fine, because `convolve_models` is just the implementation of a math operation. But I think it might be a good idea to give the users a few more examples at hand, that show how to deal with these issues in practice.

So my main comment here would be to add a more "real world" example e.g. modeling a spectral line or sky image taking a Gaussian as instrument response function and in this context shortly discuss the special properties (normalization, centering and fixed parameters of the instrument response model).